### DISCUSSION TITLE
Water or sea temperature

### DISCUSSION THREAD

**Discussion by mexysfr** [open-meteo/open-meteo#137] [Ideas]
Hi !
First of all thanks a lot for making this project available and for all the efforts made on it. This is a real pleasure to see that it's moving, almost every month.

My question is : Is there somewhere in the backlog something about getting the sea or water temperature, like in the Marine Weather API ?
I couldn't find this param anywhere, please let me know if should look twice ;)

Best !

**Comment by patrick-zippenfenig** [comment]
Not very intuitive, but for sea locations, soil temperature is the same as sea surface temperature (SST). In theory I could also add SST to the marine weather forecast API to make it more usable.

Thanks for the comment and encouraging words!

**Comment by patrick-zippenfenig** [comment]
SST is now available for testing: https://github.com/open-meteo/open-meteo/issues/407#issuecomment-2653503659

**Reply by MantraMedia** [reply]
Wow. Thank you !
